### PR TITLE
backport: 'trivial' pr10

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: contrib/linearize/example-linearize.cfg [hunk 1]
```diff
@@ -12,6 +12,11 @@ max_height=3130000
 netmagic=bf0c6bbd
 input=/home/example/.dashcore/blocks
 
+# regtest
+#netmagic=fabfb5da
+#genesis=0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206
+#input=/home/example/.bitcoin/regtest/blocks
```
Comment (UdjinM6) [inline contrib/linearize/example-linearize.cfg:18]: 17393:
```suggestion
#netmagic=fcc1b7dc
#genesis=000008ca1832a4baf228eb1553c03d3a2c8e02399550dd6ea8d65cec3ef23d2e
#input=/home/example/.dashcore/regtest/blocks
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: configure.ac [hunk 2]
```diff
@@ -735,8 +735,6 @@ if test x$ac_cv_sys_large_files != x &&
   CPPFLAGS="$CPPFLAGS -D_LARGE_FILES=$ac_cv_sys_large_files"
 fi
 
-AX_CHECK_LINK_FLAG([[-Wl,--large-address-aware]], [LDFLAGS="$LDFLAGS -Wl,--large-address-aware"])
-
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply configure.ac:740]: dropped this commit for now

File: contrib/devtools/symbol-check.py [hunk 3]
```diff
@@ -57,7 +57,6 @@
 'libgcc_s.so.1', # GCC base support
 'libc.so.6', # C library
 'libpthread.so.0', # threading
-'libanl.so.1', # DNS resolve
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply contrib/devtools/symbol-check.py:60]: Cool

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,76 @@
+name: bitcoin-core-ci
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply .github/workflows/ci.yml:1]: Okay, will open an issue related to this, as I would like better CI diversity

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**